src/objects.js

```diff
@@ -1,41 +1,85 @@
 const createPerson = (name, age) => {
   // your code here
+  return person = {
+    name:name,
+    age:age,
+  };
 };
 
 const getName = object => {
   // your code here
+  return object.name;
 };
 
 const getProperty = (property, object) => {
   // your code here
+  return object[property];
 };
 
 const hasProperty = (property, object) => {
   // your code here
+  if (object.hasOwnProperty(property) == true) {
```

Comment:
And this can be refactored similar way as I suggested at isLowerCase function in booleans.js
The logic is perfectly working here but can be a little tidier :)
The hasOwnProperty() method is returning a boolean already, you don't have to use the == equal operator, becaue essentially what you are writing is

if(true === true ) return true else return false.

You could just say: 
`````
return object.hasOwnProperty(property)
``````
, which means return true / return false.